docs/ADD_NEW_EXCHANGE.md

```diff
@@ -303,6 +303,20 @@ go run documentation.go
 
 This will generate a readme file for the exchange which can be found in the new exchange's folder
```

Comment:
One request for this file would be to remove a lot of the FTX code suggestions which are no longer relevant and could just provide general statements whilst also referring to existing implementations as a reference point

Reply:
Could you check it if I have addressed?